### DISCUSSION TITLE
`An error occurred while generating your metrics` occured when generating metrics

### DISCUSSION THREAD

**Discussion by kimlimjustin** [lowlighter/metrics#805] [Q&A] [answered]
Hey there, firstly, I'd like to thank you for your great work on `metrics`. I've used it for months on my readme profile and it works as expected. However, when I'm trying to generate my own profile today on [metrics website](https://metrics.lecoq.io/), it gives me the following error:

![image](https://user-images.githubusercontent.com/61906684/150684161-ecb1f87b-48a3-40d4-bd93-0416365e4f14.png)

any ideas bout this? thanks!

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

After looking into it brievly seems to be caused by RATE_LIMIT execeeded, the one displayed on the website may not be correct because it doesn't make difference between REST/GraphQL/search APIs

This may also be related to tokens update I performed lately, I need to check it too.
I'll look into it later today or during this week and upgrade it to latest version in the meantime 👍

**Reply by lowlighter** [reply]
Hopefully it should be fixed now 👍
